snap/snapcraft.yaml

```diff
@@ -0,0 +1,20 @@
+name: tidy-viewer
+adopt-info: tidy-viewer
+summary: Tidy Viewer (tv)
+description: |
+  Tidy Viewer (tv) is a cross-platform CLI csv pretty printer that uses column styling to maximize viewer enjoyment.
+base: core20
+
```

Comment:
Add `version: 0.0.13` and any other `mandatory` keys. https://snapcraft.io/docs/snapcraft-yaml-reference

Reply:
So, you're right: all snaps must be given a version string at build time. You can do that with the `version` key, or in this case, I sprinkled a bit of magic :tophat: in for that. `adopt-info: tidy-viewer` tells `snapcraft` to look in the `tidy-viewer` part for additional metadata. There, I used the `override-pull` key to add a step to the build that sets the snap version to the output of `git describe --tags --abbrev=0`.

This would let you automate things a bit: once you've published the snap in the store, you can point it at the Github repo, and it can [automatically build](https://snapcraft.io/build) and publish new versions to an `edge` channel. You would then just promote those builds to a `stable` channel whenever they were ready/you felt like it/you remembered. :slightly_smiling_face: 

However, it's no big deal to use a `version` key, you'd just need to manually push the version number with every new build. Let me know which approach you'd like, and if needed I can edit this PR accordingly.

Reply:
Thanks for teaching me how it works.